scripts/lib/downloadImages.ts

```diff
@@ -17,23 +17,44 @@ import { createWriteStream } from "node:fs";
 import { finished } from "stream/promises";
 import { Readable } from "stream";
 import pMap from "p-map";
+import { startWebServer, closeWebServer } from "../lib/webServer";
 
 export async function downloadImages(
   images: Array<{ src: string; dest: string }>,
+  imagesPath: string,
```

Comment:
This name is a little ambiguous: is it the images we will be writing, or the original images? Maybe use a name like `originalImagesFolderPath`? And probably have the caller put the `/artifacts` in the argument to begin with.